pkgs/top-level/all-packages.nix

```diff
@@ -10722,7 +10723,7 @@ with pkgs;
 
   clipgrab = libsForQt5.callPackage ../applications/video/clipgrab { };
 
-  cmus = callPackage ../applications/audio/cmus {
+  cmus = callPackage ../by-name/cm/cmus/package.nix {
```

Comment:
Just use `libjack2` and `ffmpeg_7` in the `package.nix` directly, then you can drop the entry here.

Reply:
for libjack2 yes, but i would like to keep the ffmpeg override so we can remove it easily on the next cmus update

Reply:
the underspecification of version pinning bothered me so much that i created an rfc https://github.com/NixOS/rfcs/pull/192